### PR TITLE
zh: docs/tasks/traffic-management/ingress/secure-ingress/

### PR DESCRIPTION
**Pull request by rootsongjc**
Mainly update the [Secure Gateways](docs/tasks/traffic-management/ingress/secure-ingress/) Chinese page.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/blog/2022/gateway-api-beta/index.md [hunk 1]
```diff
@@ -44,4 +44,4 @@ Istio 功能的其他部分，包括策略和遥测，将继续使用 Istio 特
 
 我们接下来的联合步骤是为网关 API 项目提供[增强建议](https://gateway-api.sigs.k8s.io/v1alpha2/contributing/gep/)，以确保该 API 适用于网格用例。我们已经[开始关注 API 语义](https://docs.google.com/document/d/1T_DtMQoq2tccLAtJTpo3c0ohjm25vRS35MsestSL9QU/edit)用于网格流量管理的问题，并将与在其项目中实施 Gateway API 的供应商和社区合作，以建立一个标准实施。之后，我们打算为授权和身份验证策略构建一个表示。
 
-SIG Network 作为供应商中立论坛，确保服务网格社区使用相同的语义来实现网关 API，我们期待有一个标准 API ，它适用于所有的项目，无论其技术堆栈或代理如何。
\ No newline at end of file
+SIG Network 作为供应商中立论坛，确保服务网格社区使用相同的语义来实现网关 API，我们期待有一个标准 API ，它适用于所有的项目，无论其技术堆栈或代理如何。
```
Comment (zirain) [inline content/zh/blog/2022/gateway-api-beta/index.md:47]:
```suggestion
SIG Network 作为供应商中立论坛，确保服务网格社区使用相同的语义来实现网关 API，我们期待有一个标准 API ，它适用于所有的项目，无论其使用任何技术栈或代理软件。
```

File: content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 2]
```diff
@@ -12,128 +12,262 @@ test: yes
 
 [Ingress 流量控制任务](/zh/docs/tasks/traffic-management/ingress/ingress-control)描述了如何配置入口网关以向外部公开 HTTP 服务。此任务描述如何使用 TLS 或 mTLS 公开安全的 HTTPS 服务。
 
+{{< boilerplate gateway-api-support >}}
+
 ## 准备工作{#before-you-begin}
 
-1. 执行[准备工作](/zh/docs/tasks/traffic-management/ingress/ingress-control#before-you-begin)中的步骤。完成[Ingress 流量控制](/zh/docs/tasks/traffic-management/ingress/ingress-control)中[确定 Ingress 的 IP 和端口](/zh/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports)部分任务。执行完这些步骤后，您应该已部署 Istio 和 [httpbin]({{< github_tree >}}/samples/httpbin)服务，并设置了环境变量 `INGRESS_HOST` 和 `SECURE_INGRESS_PORT` 。
+* 参考[安装指南](/zh//docs/setup/)部署 Istio。
```
Comment (zirain) [inline content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md:19]:
```suggestion
* 参考[安装指南](/zh/docs/setup/)部署 Istio。
```

**Review by zirain — CHANGES_REQUESTED** [review]
can you please checkout from master branch to reduce the number of commit for better review?

File: content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (zirain) [inline content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md:1]: prefer to split this page to a single PR

File: content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md [hunk 4]
```diff
@@ -395,77 +612,90 @@ $ kubectl create -n istio-system secret generic httpbin-credential --from-file=t
 
 ## 更多信息 {#more-info}
 
-### 密钥格式 {#key-format}
+### 秘钥格式 {#key-formats}
```
Comment (windsonsea) [inline content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md:615]:
```suggestion
### 密钥格式 {#key-formats}
```
本页其他地方用的都是 `密钥`

File: content/zh/boilerplates/gateway-api-support.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+---
+{{< tip >}}
+
+Istio 打算在[未来](/zh/blog/2022/gateway-api-beta/)将 Kubernetes Gateway API 作为流量管理的默认 API。下面的说明允许您在网格中配置流量管理时选择使用 [Gateway API](https://gateway-api.sigs.k8s.io/) 或 Istio 配置 API。根据您的偏好，按照 `Gateway API` 或 `Istio classic` 标签下的说明进行操作。
```
Comment (windsonsea) [inline content/zh/boilerplates/gateway-api-support.md:5]:
```suggestion
Istio 打算在[未来](/zh/blog/2022/gateway-api-beta/)将 Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/) 作为流量管理的默认 API。下面的说明允许您在网格中配置流量管理时选择使用 Gateway API 或 Istio 配置 API。根据您的偏好，按照 `Gateway API` 或 `Istio classic` 标签下的说明进行操作。
```
通常在第一次出现时加链接，也与 en 一致。

**Review by windsonsea — COMMENTED** [review]
Thanks for and admire your good quality about trans.

参考以下步骤同时提很多 PR：

1. 第一个 PR
  ```
  git checkout master
  git checkout -b zh-secure-gateway1
  ```
2. 提交第一个 PR 后，创建第二个 PR
  ```
  git checkout master
  git checkout -b zh-secure-gateway2
  ```

**Comment by rootsongjc** [comment]
> Thanks for and admire your good quality about trans.
> 
> 参考以下步骤同时提很多 PR：
> 
> 1. 第一个 PR
> 
> ```
> git checkout master
> git checkout -b zh-secure-gateway1
> ```
> 
> 2. 提交第一个 PR 后，创建第二个 PR
> 
> ```
> git checkout master
> git checkout -b zh-secure-gateway2
> ```

了解，只是我想一次多修改一些内容，来回提交 PR 太麻烦了，很多页面都有些小问题，我看到就一并都改了。

File: content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md (hunk 3 above)
Comment (rootsongjc) [reply content/zh/docs/tasks/traffic-management/ingress/secure-ingress/index.md:1]: 这个 PR CLA 有些问题，我关闭了重新提交吧。

**Comment by rootsongjc** [comment]
Please see the new PR: https://github.com/istio/istio.io/pull/12124

**Pull request closed without merging**